### DISCUSSION TITLE
Teslamate may affect the car's computer

### DISCUSSION THREAD

**Discussion by JacobKan** [teslamate-org/teslamate#2210] [Ideas]
Last night I shutdown my computer and of cause Teslamate. Then I go back home and charged my Tesla Model3. 

This morning I drove to office, a 3km route, and found that the "odometer since last charging" display on the car is not reset. It shows 200km which is calculated since 2days ago charging.

Does any other people find this issue?

**Comment by cwanja** [comment]
> Last night I shutdown my computer and of cause Teslamate. Then I go back home and charged my Tesla Model3.
> 
> This morning I drove to office, a 3km route, and found that the "odometer since last charging" display on the car is not reset. It shows 200km which is calculated since 2days ago charging.
> 
> Does any other people find this issue?

TeslaMate has no "write" ability to the cars computer and as such is never writing to the display or data in the main cars computer. This application is just scraping data from the non-documented Tesla API. 

What you are experiencing it sounds like is a bug in Tesla firmware. This is the second case I have heard of someone in the last week where the odometer did not reset either after a drive or a charge.

**Reply by JacobKan** [reply]
It should be true.
